### ISSUE TITLE
Iris Support

### ISSUE THREAD

**Issue by AtomSnow** [xCollateral/VulkanMod#44]
It would be nice if Iris would be supported in future

**Comment by Funtimes909** [comment]
This will literally never ever happen due to how iris and sodium use opengl graphics features

**Comment by Atemu** [comment]
I'm sure there are ways to make them compatible. VK comes from the same people as OGL, they have likely thought about keeping them compatible in some ways. I mean, there are even VK extensions that aid in *implementing* OGL on top of VK.

It probably won't be easy though.

**Comment by Etaash-mathamsetty** [comment]
> I'm sure there are ways to make them compatible. VK comes from the same people as OGL, they have likely thought about keeping them compatible in some ways. I mean, there are even VK extensions that aid in _implementing_ OGL on top of VK.
> 
> It probably won't be easy though.

the zink driver does this, it's not exactly easy to enable on windows though
zink is basically opengl -> vulkan

**Comment by Funtimes909** [comment]
> I'm sure there are ways to make them compatible. VK comes from the same people as OGL, they have likely thought about keeping them compatible in some ways. I mean, there are even VK extensions that aid in *implementing* OGL on top of VK.
> 
> 
> 
> It probably won't be easy though.

Yes but no one is going to do it for the slightly higher performance you *might* gain

**Comment by Etaash-mathamsetty** [comment]
> > I'm sure there are ways to make them compatible. VK comes from the same people as OGL, they have likely thought about keeping them compatible in some ways. I mean, there are even VK extensions that aid in _implementing_ OGL on top of VK.
> > It probably won't be easy though.
> 
> Yes but no one is going to do it for the slightly higher performance you _might_ gain

it's literally already happened (with the zink driver), and the reason the mesa project decided to make it is because manufacturers are slowly not making opengl drivers.

**Comment by RoxyCN** [comment]
You can use Pojavlauncher to run Minecraft on Amdroid device,I already succeed
![-b2c5729762c6239](https://user-images.githubusercontent.com/91813030/183601235-9138a636-0103-4551-9ba7-9ea38d8117d3.jpg)

**Comment by Felix14-v2** [comment]
> You can use Pojavlauncher to run Minecraft on Android device,I already succeed ![-b2c5729762c6239](https://user-images.githubusercontent.com/91813030/183601235-9138a636-0103-4551-9ba7-9ea38d8117d3.jpg)

What settings did you use? I get a crash at different settings, no matter how I experiment with them.

**Comment by ConfectionaryQwQ** [comment]
> > You can use Pojavlauncher to run Minecraft on Android device,I already succeed ![-b2c5729762c6239](https://user-images.githubusercontent.com/91813030/183601235-9138a636-0103-4551-9ba7-9ea38d8117d3.jpg)
> 
> What settings did you use? I get a crash at different settings, no matter how I experiment with them.

Sorry for taking so long to answer you.
1.You need to download or update the Pojavlauncher which was supposed VulanMOD (v3_vulkan mod)(Need to unzip)
Download Link:https://github.com/PojavLauncherTeam/PojavLauncher/actions/runs/2474952097
2.You should download vulkanmod which was designed for Pojavlauncher(including 2 mods,need to unzip)
Link:https://github.com/PojavLauncherTeam/VulkanMod/actions/runs/2769132000
3.Make sure you already installed Minecraft Java edition on Pojavlauncher,version1.19(Not 1.19.1or1.19.2)
4.Get Fabric loader and Fabric-API on fabricmc.net
5.Open PojavLauncher,You will see the button in the lower right corner(INSTEALL JAR.),click it and insteall Fabric 1.19
5.Put VulkanMODs in the following file path:
/storage/emulated/0/Android/data/net.kdt.pojavlaunch.debug/files/.minecraft/mods/
6.Restart PojavLauncher and play
Remember,Not all mobile GPUs support VulkanMOD,When I Use Qualcomm SnapDragon870(Adreno 650),I Failed.But I Try again on Snapdragon888(Aderno660),I succeed.
These are my experiences,I hope you can succeed.
P.S.My English is not good. Please forgive me if I make grammatical mistakes :)
